tgui/packages/tgui/interfaces/ChemHeater.js

```diff
@@ -0,0 +1,130 @@
+import { useBackend } from "../backend";
+import { AnimatedNumber, Box, Button, Dimmer, Flex, Icon, Knob, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { getTemperatureColor, getTemperatureIcon, getTemperatureChange } from './common/temperatureUtils.js';
+import { NoContainer, ReagentGraph, ReagentList } from './common/ReagentInfo.js';
+
+export const ChemHeater = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { containerData, isActive, targetTemperature } = data;
+
+  return (
+    <Window
+      title="Reagent Heater/Cooler"
+      width={320}
+      height={385}>
+      <Window.Content>
+        <ChemDisplay container={containerData} targetTemperature={targetTemperature} active={isActive} />
+        <Section title="Temperature Control">
+
+          <Stack align="center">
+            <Stack.Item>
+              <Knob
+                animated
+                size={2}
+                value={targetTemperature}
+                minValue={0}
+                maxValue={1000}
+                format={value => value + " K"}
+                onDrag={(e, value) => act('adjustTemp', { temperature: value })}
+              />
+            </Stack.Item>
+            <Stack.Item grow overflow="hidden" py={1}>
+              <Box
+                className="TemperatureNumber"
+                nowrap
+                p={1}
+                fontSize={1.5}
+                color={getTemperatureColor(targetTemperature)}
+                backgroundColor="black">
+                <Box fontSize={1}>Target</Box>
+                <Icon name={getTemperatureIcon(targetTemperature)} pr={0.5} />
+                <AnimatedNumber value={targetTemperature} /> K
+              </Box>
+            </Stack.Item>
+            <Stack.Item basis={9.6} align="center">
+              <Button
+                icon="power-off"
+                disabled={!(containerData && containerData.totalVolume)}
```

Comment:
```suggestion
                disabled={!containerData?.totalVolume}
```
Should be able to write as this instead. Please verify. These are *not* equivalent, but should be equivalent for the usage here.